cmd/lncli/walletrpc_active.go

```diff
@@ -477,6 +480,64 @@ func labelTransaction(ctx *cli.Context) error {
 	return nil
 }
 
+var publishTxCommand = cli.Command{
+	Name:      "publishtx",
+	Usage:     "Attempts to publish the passed transaction to the network.",
+	ArgsUsage: "tx_hex",
+	Description: `
+	Publish a base64-encoded raw transaction to the on-chain network. The 
```

Comment:
Why base64? The arg above mentions hex. And the bitcoind `sendrawtransaction` also takes hex.

Reply:
@guggero I wanted to emulate the behavior of /v2/wallet/tx, it only accepts base64 encoded raw transactions. I was a bit confused myself. I could change it to hex both here and at /v2/wallet/tx if you want. I think it would be good for consistency.

Reply:
Ah, the label on that API is incorrect. The gRPC API takes raw bytes. In REST that is translated to base64, but that's a property of the REST gateway/proxy. On the CLI we always use hex for things like that. So yes, please change it.

Reply:
I see. Will fix it asap.